src/components/data-table/ha-data-table.ts

```diff
@@ -338,7 +338,8 @@ export class HaDataTable extends LitElement {
                 <div class="mdc-data-table__content">
                   <div class="mdc-data-table__row" role="row">
                     <div class="mdc-data-table__cell grows center" role="cell">
-                      ${this.noDataText || "No data"}
+                      ${this.noDataText ||
+                      this.hass.localize("ui.components.data-table.no-data")}
```

Comment:
Except... we explicitly seem to translate it on use (from the same translation key).

Search for `ui.components.data-table.no-data` (e.g., ZHA panels). Do we need to remove those occurrences? Or maybe implement the caller of this one instead?

Not sure what the design idea behind this is...

Reply:
Scratch that, I think this is absolutely fine 👍 

../Frenck